invenio_drafts_resources/services/records/service.py

```diff
@@ -504,3 +504,12 @@ def _index_related_records(self, record, parent):
         #      instead (requires bulk indexing to work)
         for sibling in siblings:
             self.indexer.index(sibling)
+
```

Comment:
moderate: cannot comment on the first line 😅 , but please update copyright of CERN to 2021